src/libinjector/injector_stack.c

```diff
@@ -404,20 +404,10 @@ static bool setup_stack_64(vmi_instance_t vmi, x86_registers_t* regs, struct arg
     return 0;
 }
 
-bool setup_linux_stack(vmi_instance_t vmi, x86_registers_t* regs, struct argument args[], int nb_args)
+static bool setup_linux_syscall(vmi_instance_t vmi, x86_registers_t* regs, struct argument args[], int nb_args)
```

Comment:
Are you sure about making this function static? It won't be available from other files :thinking:

Reply:
ig windows one `setup_stack_64` is also static, we just call a wrapper around this i.e `setup_stack` which calls all these internally, so ig static shouldn't create any issues